### ISSUE TITLE
Numbers are lagging charts in queue stats display on overview page

### ISSUE THREAD

**Issue by uvzubovs** [rabbitmq/rabbitmq-management#268]
As you can see in the attached image, the number 28 msgs/sec refers to the past data point, while the new (greater) data value is available to the chart.
![capture](https://cloud.githubusercontent.com/assets/16563442/17303141/5bd3b7d2-57d4-11e6-8ed1-234795bc61c4.png)

**Comment by michaelklishin** [comment]
With what version?

**Comment by uvzubovs** [comment]
Sorry, 3.6.3

**Comment by michaelklishin** [comment]
OK, thanks. Might be addressed together with or shortly after #266.

**Comment by michaelklishin** [comment]
@dcorbacho I recall we intentionally did something similar to avoid edge cases around, ahem, edges of the chart, is this correct?

**Comment by dcorbacho** [comment]
To the best of my knowledge, this has always been the behaviour of the rates. When collecting the stats, the last datapoint is still "live" and might be receiving new samples to add to it. Thus, instant rates as the one shown on the right are calculated from the previous datapoint which is already "closed".

**Comment by michaelklishin** [comment]
@dcorbacho thanks, then it's something to consider as part of  https://github.com/rabbitmq/rabbitmq-management/issues/236.

**Comment by michaelklishin** [comment]
This may or may not be addressed in #236 but let's assume it is and file a separate issue or reopen if that's not the case in `3.6.7` (which #236 should be included into).
